.github/workflows/pull.yml

```diff
@@ -347,6 +347,9 @@ jobs:
           exit 1
         fi
 
+  android:
+    uses: ./.github/workflows/_android.yml
```

Comment:
I'm too blame here but I'm bit concern about the usage of a metal instances like `c7i.metal-24xl` to run the emulator.  That one is way over spec (and AWS doesn't provide smaller size metal instance).  And with it running in pull, it could has a negative impact on the bill.  I think let's try to switch back to a smaller instance there, `linux.4xlarge`.  The emulator test might take 10 more minute to finish IIRC, but that's probably ok.

Reply:
Seems that 4xlarge doesn't work but 12xlarge works

Reply:
Yeah, 12xlarge is ok too